modules/books.js

```diff
@@ -0,0 +1,64 @@
+import { getBookStorage, updateBookStorage, removeBookFromStorage } from './bookStorage.js';
+
+export default class Book {
+  constructor(bookheader, writer) {
+    this.bookheader = bookheader;
+    this.writer = writer;
+  }
+}
+
+export function selectItems() {
+  const bookStorage = getBookStorage();
+  let markup = '';
+
+  function removeItem() {
+    const deleteElement = [...document.getElementsByClassName('eraseItem')];
+    deleteElement.forEach((item) => {
+      item.addEventListener('click', (e) => {
+        const index = parseInt(e.target.id, 10);
+        removeBookFromStorage(index);
+        selectItems();
+      });
+    });
+  }
+
+  bookStorage.forEach((item, i) => {
+    markup += `<div class="showRw">
+      <p>"${item.bookheader}" by ${item.writer}</p>  
+      <button type="button" class="eraseItem" id=${i}>Remove</button>
+    </div>
+    <hr class="divider"/>`;
+  });
+
+  document.querySelector('.all').innerHTML = markup;
+  removeItem();
+}
+
+export function addBook(bookheader, writer) {
+  const message = document.querySelector('.displayError');
+
```

Comment:
As requested by the previous reviewer and according to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md), kindly refactor your functions to `arrow` functions. 💪